### PR TITLE
GH-693 cordova-lib@8.1.0 (minor release)

### PR DESCRIPTION
**Pull request by brodybits**

### CODE REVIEW

File: www/_posts/2018-09-17-cordova-lib-8.1.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Release Highlights
+
+The purpose of this release is to resolve `npm audit` issues, use recent releases of cordova-android and cordova-windows, and include some other recent updates.  Here are the important changes:
+
+* [GH-691](https://github.com/apache/cordova-lib/pull/691) update to use cordova-windows@~6.0.x by default
+* [GH-646](https://github.com/apache/cordova-lib/pull/646) Update **Android** platform to latest release
+* [CB-14148](https://issues.apache.org/jira/browse/CB-14148) remove nonsense "www" platform from Cordova listing ([GH-620](https://github.com/apache/cordova-lib/pull/620))
+* [CB-14243](https://issues.apache.org/jira/browse/CB-14243) change dash to underscore for save-exact key reference ([GH-631](https://github.com/apache/cordova-lib/pull/631))
+* [GH-630](https://github.com/apache/cordova-lib/pull/630) Improve plugman/uninstall.js messages
+* ([GH-628](https://github.com/apache/cordova-lib/pull/621)) Fix promise handling when removing multiple plugins
```
Comment (dpogue) [inline www/_posts/2018-09-17-cordova-lib-8.1.0.md]: There are extra parentheses around the GH-628 here

**Comment by brodybits** [comment]
Thanks @dpogue, extra parens should now be gone.

FYI I was not able to raise new JIRA issue, that is why I used GH number. I think this is not the ultimate solution since there is a potential for GH number collisions. I would like to address this another day.

**Comment by dpogue** [comment]
The ones that reference just a GH issue or pull request are fine. I'm referring to the ones that currently include both a JIRA link at the start of the sentence and a GH link at the end.

i.e.,

* [CB-14033](https://issues.apache.org/jira/browse/CB-14033) Support symbolic dir links on **Windows** ([GH-621](https://github.com/apache/cordova-lib/pull/621))

should probably just be

* [CB-14033](https://issues.apache.org/jira/browse/CB-14033) Support symbolic dir links on **Windows**

**Comment by brodybits** [comment]
Merging this PR as reviewed and approved by @janpio.

**Pull request merged**